src/Files.Uwp/Helpers/HoverDisplay/IHoverableDisplay.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Files.Uwp.Helpers.HoverDisplay
+{
+    public  interface IHoverableDisplay
```

Comment:
Why do we need this interface? By the looks of things, it's not used anywhere except for the implementation

Reply:
We use this on the sidebar as well

Reply:
@QuaintMako I was wrong, I'm realizing that the sidebar is pretty different from the tab item, and it doesn't make sense to share this interface. We'll probably create an interface for the current directory at some point, but I think we can revert this change.

Reply:
Aight! Thanks for review it guys! I'm rollbacking the change!